ios/HathorMobile.xcodeproj/project.pbxproj

```diff
@@ -512,7 +512,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = HathorMobile/HathorMobile.entitlements;
-				CURRENT_PROJECT_VERSION = 1.0.0;
+				CURRENT_PROJECT_VERSION = 1.0.1;
```

Comment:
thought: I'm not sure this would work. The way we use the `PROJECT_VERSION` property is only by manipulating the major and  versions: one for each _release candidate_ version. For example: `rc.4` would have a project version `0.4.0`.

When we have a stable release we use the `1.0.0`.
So in this case the `1.0.1` would have no meaning and should not be used.

My suggestion is that we update the full application to a new patch: `v0.27.1`. This would revert the `CURRENT_PROJECT_VERSION` to `1.0.0` as it will be a new stable.